examples/ssx-test-social/hardhat.config.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Do we need to use Hardhat on this project? I removed the dependency and the hardhat.config.ts locally and the app is still working.